unit-tests-jest/numbers.test.js

```diff
@@ -6,13 +6,23 @@ describe('evenNumbers', () => {
     const result = evenNumbers(numbers);
     expect(result).toEqual([4, 10, 0]);
   });
+  it('does not accept NAN', () => {
+    const numbers = [NaN];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([]);
+  });
 });
 
 describe('toDolars', () => {
   it('returns a number in $ format', () => {
-    const amount = 3;
+    const amount = 3.12;
+    const result = toDollars(amount);
+    expect(result).toEqual(`$${amount}`);
```

Comment:
In a unit test, you want to hard-code the value you are testing against, to make certain the function is generating the correct value. So test against `$3.12`. Below, you need to decide what you want negative values to look like and hard-code that.